CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Adds `STPPaymentMethod` protocol implementation for `STPSource` [#976](https://github.com/stripe/stripe-ios/pull/976)
```

Comment:
wdyt about: mentioning what this accomplishes? I suspect this won't mean much to our users